db/fdb_boots.c

```diff
@@ -355,13 +355,15 @@ static int _discover_remote_db_nodes(const char *dbname, const char *class,
     int rc = FDB_NOERR;
     char *node;
     cdb2_hndl_tp *db;
+    const char *comdb2dbname = "comdb2db";
 
     /* NOTE: test is dev */
     if (strncasecmp(class, "test", 4) == 0) {
         class = "dev";
+        comdb2dbname = "comdb3db";
     }
 
-    rc = cdb2_open(&db, "comdb2db", "default", 0);
+    rc = cdb2_open(&db, comdb2dbname, "default", 0);
```

Comment:
This won't work on beta and alpha.
instead of default, you can put class with its default as prod, as comdb3db exists only on dev.

Reply:
Why is "default" not working on alpha and beta CDB2 machines?

Reply:
I would have preferred to have name mapping in cdb2api, this way server could say comdb2db and talk to whatever db is behind (comdb3db in BB dev case).